platform/cert/cert_manager.go

```diff
@@ -160,6 +173,13 @@ func (c *certManager) UpdateCertificates(certs string) error {
 		return bosherr.WrapError(err, "Running command to update certificates without retries")
 	}
 
+	if strings.Compare(c.logTag, "PhotonOSCertManager") == 0 {
+		_, _, _, err = c.runner.RunCommand("cat", fmt.Sprintf("%s*.pem > /etc/pki/tls/certs/ca-bundle.crt", c.path))
```

Comment:
Can we do this natively in golang instead of shelling out to `cat`?   [`io.Copy`](https://golang.org/pkg/io/#Copy) is a native way to do what you are doing here.